README.md

```diff
@@ -274,6 +274,13 @@ Auth0
     }
 ```
 
+### Custom Domains
+
+If you are using the [Custom Domains](https://auth0.com/docs/custom-domains) feature and need to use an Auth0 endpoint
```

Comment:
It's still confusing. Now I read: "If you need to call this endpoint, use the auth0 domain". So I understand: "If I need to call `acme.com/userinfo` I need to call instead `acme12.auth0.com/userinfo` and that's not true. AFAIK using a custom domain means using my own url namespace even for authorize and userinfo calls, as long as the AUDIENCE that it's being requested (IF ANY!) is still pointing to the auth0 domain, like the example line you had before.

Also, why the new line at the end?

Reply:
It does say an 'Auth0' endpoint